### PR TITLE
blog post for drogue cloud 0.8

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+configured to allow this. As the concept existed before 0.8, no new configuration had to be added, just the APIs for the
+MQTT endpoint got extended to support this.
+
+## Static server binary
+
+Deploying a complex, cloud native stack like Drogue Cloud can be tricky. We put a lot of effort in our installer scripts
+and Helm charts to make it both easy and customizable. However, every now and then, it would be nice to "just run"
+Drogue Cloud. With 0.8, you can!
+
+Assuming you have an existing Kafka and Keycloak instance, it is as easy as:
```
Comment (lulf) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:59]: And Postgres instance!

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
++++
+title = "Drogue Cloud: Release 0.8.0"
+extra.author = "ctron"
+description = "Summer is over, Drogue Cloud 0.7.0 is out: Kafka topic provisioning, CoAP support (GSoC), Websocket integration,  TimescaleDB, and much more!"
```
Comment (lulf) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:4]: Remove duplicate description

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+~~~shell
+drogue-cloud-server --enable-all
+~~~
+
+And, it also works on Apple M1 machines. Does it bring all the features? No, it can't. There are a few that we will
+add in future releases, but we are not planning to re-invent Kubernetes. Does it run on a Raspberry Pi too?
+Probably, we didn't have the time yet to try out. But we definitely want to get there.
+
+If you are curious, you can read more about it
+[in our documentation](https://book.drogue.io/drogue-cloud/dev/deployment/bare-metal.html).
```
Comment (lulf) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:70]: There is also the last blogpost we can link to here.

File: content/2021-12-01-drogue-cloud-zero-eight/index.md (hunk 2 above)
Comment (ctron) [reply content/2021-12-01-drogue-cloud-zero-eight/index.md:4]: Whoopsie …

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+It also showcases how important the ability to integrate is. All these different projects, developed by different
+communities, can still work together to solve problems on the different layers of an IoT stack. Try it out, and maybe
+replace a component yourself.
+
+Of course, you can also cheat and just focus on the cloud side running on your laptop. Should you want to skip on buying
+a few new toys to play with.
+
+### Drogue WiFi
+
+This examples walks you through a set of steps, creating your own microcontroller firmware, measuring a temperature,
```
Comment (dejanb) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:96]: s/examples/example

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+and sending that to the cloud. Heard that before? Well sure, not only is this our prime example for the use
+case of sending telemetry data to the cloud, we also have a few other examples doing the same. Just not one that is a
+complete workshop, including a WiFi based firmware.
+
+On the cloud side, the workshop focuses on re-using an existing Drogue cloud instance. You still host the cloud
+application, the part which consumes the device data, yourself. And as always, you are also able to run Drogue Cloud
+yourself.
```
Comment (dejanb) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:103]:
```suggestion
There are two examples of how to consumer data using web sockets, a custom WebAssembly application and ready-to-use TimescaleDB/Grafana dashboard. They represent a good starting point for developing IoT applications based on Drogue Cloud
```

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+Digital twins are important as a development model for IoT application. And while we already have a brief demo with
+Eclipse Ditto, we want to expand this integration, and make it much more useful.
+
+Having metrics and tracing in the different services is a pre-requisite for scale testing. We are building this for
+scalability, so we need be able to gather some data around this. And again, this will be a bigger topic for 2022.
+
+Of course, we also have some mid-term goals. More examples and workshops is definitely on our roadmap. If you have
+interesting topics you would like us to cover, reach out! A Quarkus based self-service Web UI for our temperature demo,
+beefing up the Bluetooth gateway application for edge devices, and an OPC UA example are all things we would like to
```
Comment (dejanb) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:129]:
```suggestion
beefing up the Bluetooth gateway application for edge devices, improving application integration for the cloud in different development environments, improving off-the shelf Grafana dashboard experience and an OPC UA example are all things we would like to
```

File: content/2021-12-01-drogue-cloud-zero-eight/index.md (hunk 1 above)
Comment (jbtrystram) [reply content/2021-12-01-drogue-cloud-zero-eight/index.md:59]: It's worth mentionning that getting the aforementioned postgres, keykloack and kafka can be done as easy as `docker compose` as we provide the file.

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+And, it also works on Apple M1 machines. Does it bring all the features? No, it can't. There are a few that we will
+add in future releases, but we are not planning to re-invent Kubernetes. Does it run on a Raspberry Pi too?
+Probably, we didn't have the time yet to try out. But we definitely want to get there.
+
+If you are curious, you can read more about it
+[in our documentation](https://book.drogue.io/drogue-cloud/dev/deployment/bare-metal.html).
+
+## More workshops
+
+The idea behind out workshops is to give you concrete examples on how to use Drogue Device and Drogue Cloud. And to
```
Comment (jbtrystram) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:74]:
```suggestion
The idea behind our workshops is to give you concrete examples on how to use Drogue Device and Drogue Cloud. And to
```

File: content/2021-12-01-drogue-cloud-zero-eight/index.md [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+by someone else.
+
+## Follow-ups
+
+API keys, which we renamed to "access tokens" in 0.8, can now be used in more situations, like accessing the device
+management API. This allows you to more easily create applications, which struggle with the OAuth2 way of doing things.
+
+In 0.7 we also switched the time-series database of our example to TimescaleDB. Thanks for some help from Timescale,
+we have been able to really improve this part of the example. Many thanks for that. If you are interested, take a look. 
+
```
Comment (jbtrystram) [inline content/2021-12-01-drogue-cloud-zero-eight/index.md:115]:
```suggestion

Our cli companion tool `drg` have also been released to `0.8.0` to catch up with API changes. We also added a new subcommand : `admin` that helps you managing apps users and transfer ownership, as well as managing access tokens for all you consuming apps.` 
```

File: content/2021-12-01-drogue-cloud-zero-eight/index.md (hunk 5 above)
Comment (ctron) [reply content/2021-12-01-drogue-cloud-zero-eight/index.md:102]: In think the suggestion is off a bit. Please check if I applied that correctly.

Comment (dejanb) [reply content/2021-12-01-drogue-cloud-zero-eight/index.md:102]: Looks good

**Pull request merged**